src/sphinxcontrib/towncrier/_fragment_discovery.py

```diff
@@ -2,11 +2,14 @@
 
 
 from functools import lru_cache
+from importlib.metadata import version
 from pathlib import Path
 from typing import Optional, Set
 
 from sphinx.util import logging
 
+from packaging.version import Version
```

Comment:
I think this requires adding a dependency on `packaging` to `install_requires` in `setup.cfg`.

Reply:
That's right.

Reply:
Good catch, thanks - because I haven't added a test for this code yet, the tox run passed even with the missing dependency.

Reply:
Yeah.. The test coverage is roughly at 50% which is the primary reason I'm still using alpha versions and classifiers.